test/unit/builtins/string.spec.ts

```diff
@@ -265,3 +265,28 @@ test.each([
         `;
     expect(util.transpileAndExecute(code)).toBe(6);
 });
+
+const trimTestCases = [
+    "",
+    " ",
+    "   ",
+    "foo ",
+    "foo ",
```

Comment:
Are these 2 strings same?

Reply:
One is a tab and one is a space, but alignment makes them look the same.